src/hotspot/cpu/x86/macroAssembler_x86.cpp

```diff
@@ -10421,4 +10421,13 @@ void MacroAssembler::restore_legacy_gprs() {
   movq(rax, Address(rsp, 15 * wordSize));
   addq(rsp, 16 * wordSize);
 }
+
+void MacroAssembler::setCC(Assembler::Condition comparison, Register dst) {
```

Comment:
Generally I think we use all lowercase for assembler functions, such as `Assembler::jcc`. I think it would be easier to read if this were named `setcc` (and similar for `esetzucc`).